src/App/Mode.php

```diff
@@ -134,8 +134,15 @@ public function determine(BasePath $basepath, Database $database, Cache $configC
 	 */
 	public function determineRunMode(bool $isBackend, Module $module, array $server, MobileDetect $mobileDetect)
 	{
-		$isBackend = $isBackend ||
-		             $module->isBackend();
+		$contenttypes = ['application/jrd+json', 'application/xrd+xml', 'text/xml',
```

Comment:
Plz move the contenttypes as a constant at the beginning of the claas.

Reply:
Done